src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -748,21 +751,26 @@ private class PutCountersData
             public ByteStringContext.InternalScope KeyScope;
         }
 
-        public bool PutCounters(DocumentsOperationContext context, string documentId, string collection, string changeVector,
-            BlittableJsonReaderObject sourceData)
+        public bool PutCounters(DocumentsOperationContext context, string documentId, string collection, string changeVector, BlittableJsonReaderObject sourceData, 
```

Comment:
I do not think this is a good idea. This is error prone. Consider in the future we will add additional items here that your current and some future schema upgrades should not ever encounter.

I think it would be better to copy the method and avoid using it, especially when changes are required internally to make it actually work.

Reply:
reveret changes in CountersStorage